test/test_prototype_transforms_consistency.py

```diff
@@ -642,6 +649,74 @@ def test_call_consistency(config, args_kwargs):
     )
 
 
+@consistency_configs
+def test_jit_consistency(config, args_kwargs):
+    args, kwargs = args_kwargs
+
+    prototype_transform_eager = config.prototype_cls(*args, **kwargs)
+    legacy_transform_eager = config.legacy_cls(*args, **kwargs)
+
+    try:
+        legacy_transform_scripted = torch.jit.script(legacy_transform_eager)
+    except Exception as exc:
+        msg = str(exc)
+
+        # Some of the transform parameter variations cannot be used while scripting. For example, `Resize(size=1)`
+        # is not scriptable. One has to use `Resize(size=[1])` instead. To avoid creating a second set of
+        # parameters, we just abort the JIT call check in such a case.
+        if (
+            re.search(r"Expected a value of type 'List\[int\]' for argument '\w+' but instead found type 'int'", msg)
+            is not None
+        ):
+            return
+        if "Expected a value of type 'Union[float, int]' for argument 'fill'" in msg:
+            return
+        # In some cases trying to script the same transformation multiple times with parameters that are not supported,
+        # `torch.jit.script` will emit a cryptic error message rather than one like the ones above.
+        if "Can't redefine method: forward on class:" in msg:
+            return
+
+        # This error happens when `torch.jit.script` hits an unguarded `_log_api_usage_once`. Since that means that
+        # the transform doesn't support scripting in general, we abort the JIT call check.
+        if "'Any' object has no attribute or method '__module__'" in msg:
+            return
```

Comment:
Related to my comment just above: it makes me think we should have some tailor-made parametrization for jit support rather than being OK with JIT failure.

There is a chance that we're going to hit one of these errors message for a case that *should* pass, and we would miss it

Reply:
Yeah, that probably makes sense. I thought it were only a few that didn't work, but it turned out to be quite a few. Will send a patch.